common/dns.go

```diff
@@ -0,0 +1,82 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/miekg/dns"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func runDNS() {
+	Context("dns", func() {
+		var (
+			resolver *net.Resolver
+			values   *application.DefaultAppsValues
+		)
+
+		BeforeEach(func() {
+			values = &application.DefaultAppsValues{}
+			defaultAppsName := fmt.Sprintf("%s-default-apps", Cluster.Name)
```

Comment:
Any reason to take the value from the `default-apps-$provider` app instead of the `cluster-$provider` app? I'm afraid that if we keep moving apps from the `default-apps` app into the `cluster` app, maybe we reach a point where we don't need the `baseDomain` value in there anymore, and it'd only be present in the cluster app.

Reply:
No specific reason. I'll change it to the cluster app